src/pages/walled-gardens.md

```diff
@@ -1188,17 +2160,19 @@ Although Google by accounts does pay Igalia to contribute to Webkit to improve c
 To imply that browsers can simply contribute to WebKit negates the fact that Apple has exclusive control over the Safari WebView on iOS. For browsers on iOS what makes it into WebKit is irrelevant, it’s what functionality that is available within the Safari Webview that's important.
 
 
-#### 8.1.1. Microsoft Differentiates Edge from Chrome
-
-For example this is a list of all the features that Microsoft have [removed or replaced](https://9to5google.com/2019/04/09/chromium-edge-browser-disables-google-services/) from Chromium in Edge:
 
+#### 8.1.1. Microsoft Differentiates Edge from Chrome
 
-![List of many browser features under the title "Services we replaced or turned off"](/images/walled-gardens/34_microsoft-chrome-edge-differences.png)
+For example this is a list of all the features that Microsoft have removed or replaced from Chromium in Edge:
```

Comment:
The link in this paragraph has gone missing.

Reply:
Fixed in c33ee77